public_html/wp-content/plugins/camptix/camptix.php

```diff
@@ -3460,6 +3470,26 @@ public function get_segment( $relation, $conditions ) {
 				$post_query_conditions[] = $condition;
 				continue;
 			}
+
+			// Attended the event
+			if ( 'attended' == $condition['field'] ) {
+				$meta_query = array(
+					'key' => 'tix_attended',
+					'value' => 1,
+				);
+				switch ( $condition['op'] ) {
```

Comment:
This block might be more readable as an `if`/`else` statement - what do you think?

Reply:
I agree an `if`/`else` statement would be more readable.  I was using the same pattern I had seen for the other segments.
Here are links to those: 
* https://github.com/WordPress/wordcamp.org/blob/production/public_html/wp-content/plugins/camptix/camptix.php#L3403
* https://github.com/WordPress/wordcamp.org/blob/production/public_html/wp-content/plugins/camptix/camptix.php#L3421
* https://github.com/WordPress/wordcamp.org/blob/production/public_html/wp-content/plugins/camptix/camptix.php#L3441

Would you suggest I change the others as well?

Reply:
Yeah, I saw the others– I also think those would be more readable as if/else, but no, I wouldn't touch those in this PR. It's okay if this one looks different, the idea is to iteratively make this code better, not refactor everything all at once.